src/components/icons/CopyIcon.tsx

```diff
@@ -10,16 +10,16 @@ const CopyIcon = (props: React.SVGProps<SVGSVGElement>) => (
 		<path
 			d="M16 12.9V17.1C16 20.6 14.6 22 11.1 22H6.9C3.4 22 2 20.6 2 17.1V12.9C2 9.4 3.4 8 6.9 8H11.1C14.6 8 16 9.4 16 12.9Z"
 			stroke="#1F6EC9"
-			strokeWidth="1.5"
-			strokeLinecap="round"
-			strokeLinejoin="round"
+			stroke-width="1.5"
+			stroke-linecap="round"
```

Comment:
Why this change? Revert everything back to camel case, jsx props cannot be kebab case